src/components/UserInfo.js

```diff
@@ -11,8 +18,11 @@ export default class UserInfo {
     };
   }
 
-  setUserInfo({ name, info }) {
+  setUserInfo({ name, info, image, id }) {
```

Comment:
**Could be improved**

It's better to **destructure** the whole response from the server like that to be able to set it easily


```js

 `setUserInfo( { name, about, avatar, _id } )`
```